CONTRIBUTING.md

```diff
@@ -6,6 +6,9 @@ When opening new issues or commenting on existing issues on this repository
 please make sure discussions are related to concrete technical issues with the
 Node.js software.
 
+For general help using Node.js, please file an issue at the
+[nodejs help repository](https://github.com/nodejs/help/issues).
```

Comment:
Agree with the nit. It's `Node.js` in documentation and other prose so we should use that here.